internal/chain/ws_client.go

```diff
@@ -27,6 +29,7 @@ func NewWsClient(url string) (*WsClient, error) {
 func (client *WsClient) GetBlockByNumber(number uint64) (blk *Block, err error) {
 	blk = new(Block)
 	err = client.rpc.Call(blk, "eth_getBlockByNumber", fmt.Sprintf("0x%x", number), false)
+	metrics.RpcCallsCounter.Inc()
```

Comment:
Right now it looks like `GetBlockByNumberCounter`. You should add this to all RPC methods I think

Reply:
Yeah. That was an intention of this metric, to capture all rpc calls. Just forgot to add to  all methods. Fixed now.